packages/console/src/Components/Analysis/Edit/AnalysisEdit.tsx

```diff
@@ -37,9 +37,6 @@ function AnalysisEdit() {
   const initialData = useRef<IAnalysis>({} as IAnalysis);
   const loading = !data.id;
 
-  /**
-   * Should return if the initial data is different from the current data.
```

Comment:
Removed the obvious/redundant comments, but only in files I'm already modifying for a reason

Reply:
These honestly make the code harder to read since there's too much noise